test/basic.js

```diff
@@ -49,7 +49,7 @@ function createDefaultsTest (combination) {
         resourcesPath = path.join(finalPath, util.generateResourcesPath(opts))
         fs.stat(path.join(finalPath, generateNamePath(opts)), cb)
       }, function (stats, cb) {
-        if (opts.platform === 'darwin') {
+        if (opts.platform === 'darwin' || opts.platform === 'mas') {
```

Comment:
I keep seeing variations on this idiom in the code and part of me wants to refactor it to

```javascript
if (platform_is_mac(opts.platform)) {
```